truncate/table_schema_test.go

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package truncate
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+func TestFilterTableSchemas(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]struct {
+		schemas       []*tableSchema
+		excludeTables []string
+		want          []*tableSchema
+	}{
+		"Exclude the parent tables by tracing up to the topmost level.": {
```

Comment:
To have a consistent style with existing tests, can we use `desc` (description) field to describe a test?

ref: https://github.com/cloudspannerecosystem/spanner-truncate/blob/main/truncate/coordinator_test.go#L34

Reply:
Sure! I'll rewrite it in the same format.